extensions/arc/deployment/src/test/java/io/quarkus/arc/test/config/nativebuild/NativeBuildConfigInjectionOkTest.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.arc.test.config.nativebuild;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusProdModeTest;
+
+public class NativeBuildConfigInjectionOkTest {
+
+    @RegisterExtension
+    static final QuarkusProdModeTest TEST = new QuarkusProdModeTest()
+            .setBuildNative(true)
+            .withApplicationRoot(
+                    root -> root.addClass(Ok.class));
+
+    @Test
+    void test() {
+    }
+}
```

Comment:
I suppose you didn't add a test for "runtime-fixed" configuration properties because there is none available in the "Arc" extension?

Reply:
Yes.